tests/pytest/test_regression.py

```diff
@@ -93,11 +93,14 @@ def run_model(rundir, model_image):
     docker_run = ['docker', 'run', '--rm']
     rundir_abs = os.path.abspath(rundir)
     rundir_mount = ['-v', f'{rundir_abs}:' + docker_runpath + '/rundir']
+    data_abs = os.path.abspath(os.path.join(rundir_abs, 'test_data'))
+    os.makedirs(data_abs)
+    data_mount = ['-v', f'{data_abs}:' + docker_runpath + '/test_data']
```

Comment:
If I'm reading this correctly, in the docker container the test data directory is a sibling of the rundir, while on the host it is a child of the rundir? Just wondering if there's any particular reason for the test data move.

Reply:
When I was generating the test data separately, I thought it made more sense as a sibling, but then that doesn't work with the path structure of the tests here. I'll change it to make them consistent. Thanks for bringing it up!